core/config/config.go

```diff
@@ -401,6 +404,17 @@ type ScrubbingConfig struct {
 	ScrubEligibleDuration time.Duration
 }
 
+type StreamReconciliationConfig struct {
+	WorkerPoolSize int // If 0, default to 8.
+}
+
+func (s *StreamReconciliationConfig) GetWorkerPoolSize() int {
```

Comment:
no need for this function:
set it to 8 in GetDefaultConfig();
if setting is set, it will overwritten.